enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -95,9 +95,12 @@ export const MetabaseProviderInternal = ({
   );
 };
 
-export const MetabaseProvider = memo(function MetabaseProvider(
-  props: MetabaseProviderProps,
-) {
+export const MetabaseProvider = memo(function MetabaseProvider({
+  // @ts-expect-error -- we don't want to expose the store prop
+  // eslint-disable-next-line react/prop-types
+  store = getSdkStore(),
```

Comment:
This makes it so the store is not a singleton, so on storybook it's a fresh instance on each story.
Otherwise the auth state was shared across stories making the order in which you opened them change the behaviour 😅